library/admin/src/Service/Globals.php

```diff
@@ -0,0 +1,254 @@
+<?php
+/**
+ * This file is part of OpenEMR.
+ *
+ * @link https://github.com/openemr/openemr/tree/master
+ * @license https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
```

Comment:
this needs a full license/copyright/author header

Reply:
Also, shouldn't we be avoiding caps in file/dir names (except for class files of course)?

Reply:
@robertdown ,
Also, see my comments directly above (i just realized this is my PR and you don't get these comments automatically)